pkgs/by-name/de/deadlock-modmanager/package.nix

```
@@ -0,0 +1,135 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  cargo-tauri,
+  nodejs,
+  pnpm_9,
+  pkg-config,
+  wrapGAppsHook3,
+  desktop-file-utils,
+  webkitgtk_4_1,
+  cairo,
+  gdk-pixbuf,
+  glib,
+  glib-networking,
+  gtk3,
+  libsoup_3,
+  pango,
+  openssl,
+  bzip2,
+  gst_all_1,
+  makeDesktopItem,
+  fontconfig,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "deadlock-modmanager";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "Stormix";
+    repo = "deadlock-modmanager";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-VUiAZxWv6Pt291F4lp0ouafFzB3KwdRrF1niURa/+/s=";
+  };
```

```suggestion
  src = fetchFromGitHub {
    owner = "deadlock-mod-manager";
    repo = "deadlock-mod-manager";
    tag = "v${finalAttrs.version}";
    hash = "sha256-VUiAZxWv6Pt291F4lp0ouafFzB3KwdRrF1niURa/+/s=";
  };
```
https://github.com/stormix/deadlock-modmanager redirects to this repo